main.go

```diff
@@ -1,15 +1,25 @@
 package main
 
 import (
+	"flag"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/codilime/contrail-windows-docker/driver"
 	"github.com/docker/go-plugins-helpers/network"
 	"github.com/docker/go-plugins-helpers/sdk"
 )
 
 func main() {
-	d, err := driver.NewDriver()
-	if err != nil {
+	var subnet = flag.String("subnet", "172.117.0.0/16", "subnet in CIDR format for HNS")
+	var gateway = flag.String("gateway", "172.117.0.1", "default gateway IP for HNS")
+	var adapter = flag.String("adapter", "Ethernet0",
```

Comment:
What is `Ethernet0`? `Get-NetAdapter | format-custom` doesn't contain such a string, how do i get it for my desired adapter?